### PR TITLE
Create preset overlay settings sample

### PR DESCRIPTION
**Pull request by PlamenaMiteva**

### CODE REVIEW

File: src/app/interactions/interactions-routing.module.ts [hunk 1]
```diff
@@ -247,6 +248,11 @@ export const interactionsRoutes: Routes = [
         data: interactionsRoutesData["overlay-scroll-sample-2"],
         path: "overlay-scroll-sample-2"
     },
+    {
+        component: OverlayPresetSettingsSampleComponent,
+        data: interactionsRoutesData["overlay-relative-presets-sample"],
```
Comment (wnvko) [inline src/app/interactions/interactions-routing.module.ts:253]: the new sample was added with key `overlay-preset-settings-sample` in `interactionsRoutesData`. The key here is wrong.
```suggestion
        data: interactionsRoutesData["overlay-preset-settings-sample"],
```

**Review by wnvko — CHANGES_REQUESTED** [review]
The new sample should be added to `OverlayConfigGenerator.ts`.

**Review by wnvko — CHANGES_REQUESTED** [review]
The new sample should be added to `OverlayConfigGenerator.ts`.

**Comment by Lipata** [comment]
@PlamenaMiteva, please create this pull request only from vNext branch.

**Pull request closed without merging**